packages/components/sidebar/src/sidebar-item.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <div
+    v-if="isMenuItem"
+    class="puik-sidebar-item"
+    :class="{ 'puik-sidebar-item--active': active }"
+  >
+    <puik-button
+      :aria-label="title"
+      fluid
+      variant="text"
+      class="puik-sidebar-item__button"
+      :to="props.to"
+      :href="props.href"
+    >
+      <span>{{ title }}</span>
+    </puik-button>
+  </div>
+  <div
+    v-else
+    class="puik-sidebar-item"
+    :class="{ 'puik-sidebar-item--active': active }"
+  >
+    <puik-button
+      :aria-label="title"
+      fluid
+      variant="text"
+      class="puik-sidebar-item__button"
+      :left-icon="icon"
+      :to="props.to"
+      :href="props.href"
+    >
+      <span v-if="sidebarValues?.extended.value">{{ title }}</span>
+    </puik-button>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { inject, computed } from 'vue'
+import PuikButton from '../../button'
+import { sidebarKey, groupItemKey } from './sidebar'
+import { sidebarItemProps } from './sidebar-item'
+
+defineOptions({
+  name: 'PuikSidebarItem',
+})
+
+const props = defineProps(sidebarItemProps)
```

Comment:
props n'est pas utilisé autre part que dans le template

```suggestion
defineProps(sidebarItemProps)
```